src/scenarios/tx_flood.py

```diff
@@ -7,24 +10,61 @@
 
```

Comment:
Perhaps you could remove 

```
BLOCKS = 100
TXS = 100
```

While you're in here? (unused?)

Reply:
nice thanks will do